include/mra/tensor/tensor.h

```diff
@@ -89,6 +89,70 @@ namespace mra {
 
     Tensor& operator=(const Tensor<T, NDIM, Allocator>& other) = delete;
 
+    Tensor operator+(const Tensor<T, NDIM, Allocator>& other) const {
+      if (this->size() != other.size()) {
+        throw std::runtime_error("Cannot add tensors of different sizes.");
+      }
+      Tensor result(*this);
+      for(size_type i = 0; i < this->size(); ++i) {
+        result.data()[i] += other.data()[i];
+      }
+      return result;
+    }
```

Comment:
This is not a good idea. We cannot do this on the device so we should not allow it in the API (even if you only use it on the host)

Reply:
This cannot go in. Please remove and adjust the places where it's used to use TensorView instead.